packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -154,6 +163,12 @@ export default {
     autoFocus: {
       control: 'boolean',
       defaultValue: false
+    },
+    isOpen: {
+      control: 'boolean'
+    },
+    defaultOpen: {
```

Comment:
I think you need to leave this one as a story since it's an initial state type of thing

Reply:
nevermind, you can get it as a refresh

Reply:
The autofocus is also on refresh.